packages/react-graphql/src/hooks/query.ts

```diff
@@ -38,12 +38,16 @@ export default function useQuery<
   } = options;
   const client = useApolloClient(overrideClient);
 
-  if (typeof window === 'undefined' && skip) {
+  if ((typeof window === 'undefined' && skip) || fetchPolicy === 'no-cache') {
     return createDefaultResult(client, variables);
   }
 
   const query = useGraphQLDocument(queryOrComponent);
 
+  const normalizedFetchPolicy =
+    typeof window === 'undefined' && fetchPolicy === 'network-only'
+      ? 'cache-first'
```

Comment:
A test explaining this behaviour would be grand.

Reply:
I'm a little conflicted, because it's actually very hard to test this given that Jest's default environment includes the DOM.

Reply:
A few Quilt tests use `@jest-environment node`.  Is that not possible here?

Reply:
Hit is, and I did that. I was just being lazy before, thanks for calling it out.